wazo_confd/helpers/common.py

```diff
@@ -42,6 +42,7 @@ def wrapper(*args, **kwargs):
                 logger.error("%s: %s", error, error.details, exc_info=True)
             return [str(error)], error.status_code
         except ProvdError as error:
+            logger.exception(str(error))
```

Comment:
Are you positive that all `ProvdError` are code errors and should print a Traceback?

Reply:
No, but I think it's better to always log something when there's an unexpected error and then filter out those we don't care about if it's problematic, than fail silently. 
If it was expected, a comment would have been helpful.

Reply:
I could avoid the traceback and only log the error though.